omnigibson/systems/micro_particle_system.py

```diff
@@ -43,6 +44,7 @@
 
 # TODO: Tune these default values!
 # TODO (eric): figure out whether one offset can fit all
+m.MAX_CLOTH_PARTICLES = 20000
```

Comment:
is there a link to their doc that specify this magic number?

Reply:
The actual magic number is something around 22600 or something. It shows up in an error when you exceed it with GPU pipeline on, docs search doesnt point at anything. I left a bit of margin too.